Use of Metafacture master-SNAPSHOT breaks project tests and web tests

I locally published the current Metafacture-Core master-SNAPSHOT to use it in lobid-resources. This seems to be an additional problem with regard to the failing actions #2348 . The current master of metafacture-core has multiple updates with regard to dependencies that create transitive problems.


```
$ mvn clean install -DskipTests=false -DgenerateTestData=true
[INFO] Scanning for projects...
[INFO] 
[INFO] ---------------------< org.lobid:lobid-resources >----------------------
[INFO] Building lobid-resources 1.0.2-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from central: https://repo.maven.apache.org/maven2/commons-io/commons-io/2.21.0/commons-io-2.21.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/commons-io/commons-io/2.21.0/commons-io-2.21.0.pom (21 kB at 32 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/91/commons-parent-91.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/91/commons-parent-91.pom (79 kB at 488 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/apache/35/apache-35.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/apache/35/apache-35.pom (24 kB at 238 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.13.4/junit-bom-5.13.4.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.13.4/junit-bom-5.13.4.pom (5.7 kB at 62 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.28.0/commons-compress-1.28.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.28.0/commons-compress-1.28.0.pom (25 kB at 288 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/85/commons-parent-85.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/85/commons-parent-85.pom (78 kB at 799 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.13.1/junit-bom-5.13.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.13.1/junit-bom-5.13.1.pom (5.6 kB at 58 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.19.0/commons-codec-1.19.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.19.0/commons-codec-1.19.0.pom (18 kB at 175 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-io/commons-io/2.20.0/commons-io-2.20.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/commons-io/commons-io/2.20.0/commons-io-2.20.0.pom (20 kB at 201 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-lang3/3.18.0/commons-lang3-3.18.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-lang3/3.18.0/commons-lang3-3.18.0.pom (32 kB at 297 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.21.1/jackson-core-2.21.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.21.1/jackson-core-2.21.1.pom (10 kB at 100 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-base/2.21.1/jackson-base-2.21.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-base/2.21.1/jackson-base-2.21.1.pom (13 kB at 126 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.21.1/jackson-bom-2.21.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.21.1/jackson-bom-2.21.1.pom (20 kB at 235 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-parent/2.21/jackson-parent-2.21.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-parent/2.21/jackson-parent-2.21.pom (6.9 kB at 72 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/75/oss-parent-75.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/oss-parent/75/oss-parent-75.pom (24 kB at 241 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.14.1/junit-bom-5.14.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.14.1/junit-bom-5.14.1.pom (5.7 kB at 56 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.21.1/jackson-databind-2.21.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.21.1/jackson-databind-2.21.1.pom (23 kB at 217 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.21/jackson-annotations-2.21.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.21/jackson-annotations-2.21.pom (7.4 kB at 74 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/jayway/jsonpath/json-path/2.10.0/json-path-2.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/jayway/jsonpath/json-path/2.10.0/json-path-2.10.0.pom (1.9 kB at 20 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/net/minidev/json-smart/2.6.0/json-smart-2.6.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/net/minidev/json-smart/2.6.0/json-smart-2.6.0.pom (12 kB at 116 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/net/minidev/accessors-smart/2.6.0/accessors-smart-2.6.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/net/minidev/accessors-smart/2.6.0/accessors-smart-2.6.0.pom (14 kB at 143 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/ow2/asm/asm/9.7.1/asm-9.7.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/ow2/asm/asm/9.7.1/asm-9.7.1.pom (2.4 kB at 24 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/erosb/everit-json-schema/1.14.6/everit-json-schema-1.14.6.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/erosb/everit-json-schema/1.14.6/everit-json-schema-1.14.6.pom (13 kB at 132 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/json/json/20250107/json-20250107.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/json/json/20250107/json-20250107.pom (8.7 kB at 102 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-validator/commons-validator/1.9.0/commons-validator-1.9.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/commons-validator/commons-validator/1.9.0/commons-validator-1.9.0.pom (16 kB at 167 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/re2j/re2j/1.8/re2j-1.8.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/re2j/re2j/1.8/re2j-1.8.pom (1.8 kB at 15 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/xalan/serializer/2.7.3/serializer-2.7.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/xalan/serializer/2.7.3/serializer-2.7.3.pom (389 B at 4.5 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/xalan/xalan/2.7.3/xalan-2.7.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/xalan/xalan/2.7.3/xalan-2.7.3.pom (384 B at 4.3 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/opencsv/opencsv/5.12.0/opencsv-5.12.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/opencsv/opencsv/5.12.0/opencsv-5.12.0.pom (37 kB at 339 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.13.2/junit-bom-5.13.2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.13.2/junit-bom-5.13.2.pom (5.7 kB at 64 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-text/1.13.1/commons-text-1.13.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-text/1.13.1/commons-text-1.13.1.pom (23 kB at 237 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/81/commons-parent-81.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/81/commons-parent-81.pom (78 kB at 686 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/apache/33/apache-33.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/apache/33/apache-33.pom (24 kB at 272 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.11.4/junit-bom-5.11.4.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.11.4/junit-bom-5.11.4.pom (5.6 kB at 51 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-lang3/3.17.0/commons-lang3-3.17.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-lang3/3.17.0/commons-lang3-3.17.0.pom (31 kB at 303 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/73/commons-parent-73.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/73/commons-parent-73.pom (78 kB at 612 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.11.0/junit-bom-5.11.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.11.0/junit-bom-5.11.0.pom (5.6 kB at 61 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-beanutils/commons-beanutils/1.11.0/commons-beanutils-1.11.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/commons-beanutils/commons-beanutils/1.11.0/commons-beanutils-1.11.0.pom (19 kB at 186 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/84/commons-parent-84.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/84/commons-parent-84.pom (78 kB at 646 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/apache/34/apache-34.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/apache/34/apache-34.pom (24 kB at 292 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.12.2/junit-bom-5.12.2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/junit/junit-bom/5.12.2/junit-bom-5.12.2.pom (5.6 kB at 65 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-logging/commons-logging/1.3.5/commons-logging-1.3.5.pom
Downloaded from central: https://repo.maven.apache.org/maven2/commons-logging/commons-logging/1.3.5/commons-logging-1.3.5.pom (32 kB at 354 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-collections4/4.5.0/commons-collections4-4.5.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-collections4/4.5.0/commons-collections4-4.5.0.pom (26 kB at 260 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-logging/commons-logging/1.3.2/commons-logging-1.3.2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/commons-logging/commons-logging/1.3.2/commons-logging-1.3.2.pom (32 kB at 363 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/xtext-dev-bom/2.37.0/xtext-dev-bom-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/xtext-dev-bom/2.37.0/xtext-dev-bom-2.37.0.pom (17 kB at 177 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-arq/4.10.0/jena-arq-4.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-arq/4.10.0/jena-arq-4.10.0.pom (7.5 kB at 91 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena/4.10.0/jena-4.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena/4.10.0/jena-4.10.0.pom (41 kB at 482 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.15.3/jackson-bom-2.15.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.15.3/jackson-bom-2.15.3.pom (18 kB at 228 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/jetty/jetty-bom/10.0.17/jetty-bom-10.0.17.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/jetty/jetty-bom/10.0.17/jetty-bom-10.0.17.pom (19 kB at 209 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-core/4.10.0/jena-core-4.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-core/4.10.0/jena-core-4.10.0.pom (8.7 kB at 92 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-base/4.10.0/jena-base-4.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-base/4.10.0/jena-base-4.10.0.pom (5.1 kB at 53 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-csv/1.10.0/commons-csv-1.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-csv/1.10.0/commons-csv-1.10.0.pom (20 kB at 205 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.24.0/commons-compress-1.24.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.24.0/commons-compress-1.24.0.pom (22 kB at 242 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/61/commons-parent-61.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-parent/61/commons-parent-61.pom (81 kB at 830 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-iri/4.10.0/jena-iri-4.10.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-iri/4.10.0/jena-iri-4.10.0.pom (3.2 kB at 36 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/roaringbitmap/RoaringBitmap/1.0.0/RoaringBitmap-1.0.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/roaringbitmap/RoaringBitmap/1.0.0/RoaringBitmap-1.0.0.pom (2.1 kB at 24 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/jsonld-java/jsonld-java/0.13.4/jsonld-java-0.13.4.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/jsonld-java/jsonld-java/0.13.4/jsonld-java-0.13.4.pom (6.1 kB at 57 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/jsonld-java/jsonld-java-parent/0.13.4/jsonld-java-parent-0.13.4.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/jsonld-java/jsonld-java-parent/0.13.4/jsonld-java-parent-0.13.4.pom (15 kB at 166 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.11.4/jackson-bom-2.11.4.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-bom/2.11.4/jackson-bom-2.11.4.pom (15 kB at 150 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/httpcomponents/httpclient-cache/4.5.14/httpclient-cache-4.5.14.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/httpcomponents/httpclient-cache/4.5.14/httpclient-cache-4.5.14.pom (6.6 kB at 66 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/slf4j/jcl-over-slf4j/1.7.36/jcl-over-slf4j-1.7.36.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/slf4j/jcl-over-slf4j/1.7.36/jcl-over-slf4j-1.7.36.pom (1.2 kB at 12 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.15.3/jackson-core-2.15.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.15.3/jackson-core-2.15.3.pom (9.8 kB at 115 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-base/2.15.3/jackson-base-2.15.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jackson-base/2.15.3/jackson-base-2.15.3.pom (11 kB at 127 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.15.3/jackson-databind-2.15.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-databind/2.15.3/jackson-databind-2.15.3.pom (19 kB at 205 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.15.3/jackson-annotations-2.15.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-annotations/2.15.3/jackson-annotations-2.15.3.pom (7.1 kB at 77 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/apicatalog/titanium-json-ld/1.3.2/titanium-json-ld-1.3.2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/apicatalog/titanium-json-ld/1.3.2/titanium-json-ld-1.3.2.pom (1.6 kB at 16 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/glassfish/jakarta.json/2.0.1/jakarta.json-2.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/glassfish/jakarta.json/2.0.1/jakarta.json-2.0.1.pom (12 kB at 131 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/glassfish/json/2.0.1/json-2.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/glassfish/json/2.0.1/json-2.0.1.pom (19 kB at 191 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-java/3.24.3/protobuf-java-3.24.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-java/3.24.3/protobuf-java-3.24.3.pom (1.6 kB at 16 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-parent/3.24.3/protobuf-parent-3.24.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-parent/3.24.3/protobuf-parent-3.24.3.pom (9.0 kB at 96 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-bom/3.24.3/protobuf-bom-3.24.3.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-bom/3.24.3/protobuf-bom-3.24.3.pom (3.7 kB at 42 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/thrift/libthrift/0.19.0/libthrift-0.19.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/thrift/libthrift/0.19.0/libthrift-0.19.0.pom (2.7 kB at 29 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase/2.37.0/org.eclipse.xtext.xbase-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase/2.37.0/org.eclipse.xtext.xbase-2.37.0.pom (1.1 kB at 4.2 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/xtext-parent/2.37.0/xtext-parent-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/xtext-parent/2.37.0/xtext-parent-2.37.0.pom (28 kB at 273 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.common.types/2.37.0/org.eclipse.xtext.common.types-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.common.types/2.37.0/org.eclipse.xtext.common.types-2.37.0.pom (1.0 kB at 2.0 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext/2.37.0/org.eclipse.xtext-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext/2.37.0/org.eclipse.xtext-2.37.0.pom (2.6 kB at 27 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib/2.37.0/org.eclipse.xtend.lib-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib/2.37.0/org.eclipse.xtend.lib-2.37.0.pom (1.2 kB at 12 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase.lib/2.37.0/org.eclipse.xtext.xbase.lib-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase.lib/2.37.0/org.eclipse.xtext.xbase.lib-2.37.0.pom (725 B at 8.1 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/33.3.1-jre/guava-33.3.1-jre.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/33.3.1-jre/guava-33.3.1-jre.pom (9.3 kB at 109 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/33.3.1-jre/guava-parent-33.3.1-jre.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/33.3.1-jre/guava-parent-33.3.1-jre.pom (20 kB at 246 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/failureaccess/1.0.2/failureaccess-1.0.2.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/failureaccess/1.0.2/failureaccess-1.0.2.pom (3.3 kB at 42 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/checkerframework/checker-qual/3.43.0/checker-qual-3.43.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/checkerframework/checker-qual/3.43.0/checker-qual-3.43.0.pom (2.1 kB at 25 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_annotations/2.28.0/error_prone_annotations-2.28.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_annotations/2.28.0/error_prone_annotations-2.28.0.pom (4.3 kB at 51 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_parent/2.28.0/error_prone_parent-2.28.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_parent/2.28.0/error_prone_parent-2.28.0.pom (13 kB at 143 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/j2objc/j2objc-annotations/3.0.0/j2objc-annotations-3.0.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/j2objc/j2objc-annotations/3.0.0/j2objc-annotations-3.0.0.pom (5.1 kB at 57 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib.macro/2.37.0/org.eclipse.xtend.lib.macro-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib.macro/2.37.0/org.eclipse.xtend.lib.macro-2.37.0.pom (864 B at 8.8 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.util/2.37.0/org.eclipse.xtext.util-2.37.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.util/2.37.0/org.eclipse.xtext.util-2.37.0.pom (1.7 kB at 19 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/ch/qos/reload4j/reload4j/1.2.25/reload4j-1.2.25.pom
Downloaded from central: https://repo.maven.apache.org/maven2/ch/qos/reload4j/reload4j/1.2.25/reload4j-1.2.25.pom (16 kB at 163 kB/s)
Downloading from sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/emf/org.eclipse.emf.common/maven-metadata.xml
Downloading from central: https://repo.maven.apache.org/maven2/com/google/inject/guice/7.0.0/guice-7.0.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/inject/guice/7.0.0/guice-7.0.0.pom (9.8 kB at 110 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/inject/guice-parent/7.0.0/guice-parent-7.0.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/inject/guice-parent/7.0.0/guice-parent-7.0.0.pom (19 kB at 180 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/jakarta/inject/jakarta.inject-api/2.0.1/jakarta.inject-api-2.0.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/jakarta/inject/jakarta.inject-api/2.0.1/jakarta.inject-api-2.0.1.pom (5.9 kB at 67 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava/31.0.1-jre/guava-31.0.1-jre.pom (11 kB at 109 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/31.0.1-jre/guava-parent-31.0.1-jre.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/guava/guava-parent/31.0.1-jre/guava-parent-31.0.1-jre.pom (14 kB at 132 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/checkerframework/checker-qual/3.12.0/checker-qual-3.12.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/checkerframework/checker-qual/3.12.0/checker-qual-3.12.0.pom (2.1 kB at 23 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_annotations/2.7.1/error_prone_annotations-2.7.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_annotations/2.7.1/error_prone_annotations-2.7.1.pom (2.1 kB at 24 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_parent/2.7.1/error_prone_parent-2.7.1.pom
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/errorprone/error_prone_parent/2.7.1/error_prone_parent-2.7.1.pom (7.0 kB at 79 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.equinox.common/3.17.100/org.eclipse.equinox.common-3.17.100.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.equinox.common/3.17.100/org.eclipse.equinox.common-3.17.100.pom (1.6 kB at 16 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.osgi/3.18.300/org.eclipse.osgi-3.18.300.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.osgi/3.18.300/org.eclipse.osgi-3.18.300.pom (1.3 kB at 14 kB/s)
Downloading from sonatype-snapshots: https://oss.sonatype.org/content/repositories/snapshots/org/eclipse/emf/org.eclipse.emf.ecore/maven-metadata.xml
Downloading from central: https://repo.maven.apache.org/maven2/io/github/classgraph/classgraph/4.8.179/classgraph-4.8.179.pom
Downloaded from central: https://repo.maven.apache.org/maven2/io/github/classgraph/classgraph/4.8.179/classgraph-4.8.179.pom (30 kB at 293 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-io/commons-io/2.21.0/commons-io-2.21.0.jar
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.28.0/commons-compress-1.28.0.jar
Downloading from central: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.19.0/commons-codec-1.19.0.jar
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-lang3/3.18.0/commons-lang3-3.18.0.jar
Downloading from central: https://repo.maven.apache.org/maven2/com/jayway/jsonpath/json-path/2.10.0/json-path-2.10.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/commons-io/commons-io/2.21.0/commons-io-2.21.0.jar (585 kB at 2.9 MB/s)
Downloading from central: https://repo.maven.apache.org/maven2/net/minidev/json-smart/2.6.0/json-smart-2.6.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/net/minidev/json-smart/2.6.0/json-smart-2.6.0.jar (123 kB at 363 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/net/minidev/accessors-smart/2.6.0/accessors-smart-2.6.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/commons-codec/commons-codec/1.19.0/commons-codec-1.19.0.jar (375 kB at 884 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/github/erosb/everit-json-schema/1.14.6/everit-json-schema-1.14.6.jar
Downloaded from central: https://repo.maven.apache.org/maven2/net/minidev/accessors-smart/2.6.0/accessors-smart-2.6.0.jar (30 kB at 68 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/json/json/20250107/json-20250107.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-compress/1.28.0/commons-compress-1.28.0.jar (1.1 MB at 2.2 MB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/re2j/re2j/1.8/re2j-1.8.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/jayway/jsonpath/json-path/2.10.0/json-path-2.10.0.jar (279 kB at 518 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/xalan/serializer/2.7.3/serializer-2.7.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/json/json/20250107/json-20250107.jar (81 kB at 144 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/xalan/xalan/2.7.3/xalan-2.7.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-lang3/3.18.0/commons-lang3-3.18.0.jar (703 kB at 1.2 MB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/opencsv/opencsv/5.12.0/opencsv-5.12.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/github/erosb/everit-json-schema/1.14.6/everit-json-schema-1.14.6.jar (253 kB at 426 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-text/1.13.1/commons-text-1.13.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/re2j/re2j/1.8/re2j-1.8.jar (114 kB at 189 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-collections4/4.5.0/commons-collections4-4.5.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/xalan/serializer/2.7.3/serializer-2.7.3.jar (284 kB at 309 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-arq/4.10.0/jena-arq-4.10.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/opencsv/opencsv/5.12.0/opencsv-5.12.0.jar (242 kB at 256 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/httpcomponents/httpclient-cache/4.5.14/httpclient-cache-4.5.14.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-text/1.13.1/commons-text-1.13.1.jar (259 kB at 238 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/apicatalog/titanium-json-ld/1.3.2/titanium-json-ld-1.3.2.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-collections4/4.5.0/commons-collections4-4.5.0.jar (899 kB at 797 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/glassfish/jakarta.json/2.0.1/jakarta.json-2.0.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/httpcomponents/httpclient-cache/4.5.14/httpclient-cache-4.5.14.jar (164 kB at 124 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-java/3.24.3/protobuf-java-3.24.3.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/glassfish/jakarta.json/2.0.1/jakarta.json-2.0.1.jar (144 kB at 102 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/thrift/libthrift/0.19.0/libthrift-0.19.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/apicatalog/titanium-json-ld/1.3.2/titanium-json-ld-1.3.2.jar (291 kB at 205 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-core/4.10.0/jena-core-4.10.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/xalan/xalan/2.7.3/xalan-2.7.3.jar (3.5 MB at 2.2 MB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-base/4.10.0/jena-base-4.10.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-arq/4.10.0/jena-arq-4.10.0.jar (3.8 MB at 2.3 MB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-csv/1.10.0/commons-csv-1.10.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/thrift/libthrift/0.19.0/libthrift-0.19.0.jar (360 kB at 210 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-iri/4.10.0/jena-iri-4.10.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-base/4.10.0/jena-base-4.10.0.jar (256 kB at 146 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/commons-cli/commons-cli/1.5.0/commons-cli-1.5.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/commons/commons-csv/1.10.0/commons-csv-1.10.0.jar (55 kB at 30 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/roaringbitmap/RoaringBitmap/1.0.0/RoaringBitmap-1.0.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-iri/4.10.0/jena-iri-4.10.0.jar (139 kB at 75 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/emf/org.eclipse.emf.ecore/2.37.0/org.eclipse.emf.ecore-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/commons-cli/commons-cli/1.5.0/commons-cli-1.5.0.jar (58 kB at 31 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase/2.37.0/org.eclipse.xtext.xbase-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/roaringbitmap/RoaringBitmap/1.0.0/RoaringBitmap-1.0.0.jar (448 kB at 227 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.common.types/2.37.0/org.eclipse.xtext.common.types-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/protobuf/protobuf-java/3.24.3/protobuf-java-3.24.3.jar (1.8 MB at 807 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/ow2/asm/asm/9.7.1/asm-9.7.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/emf/org.eclipse.emf.ecore/2.37.0/org.eclipse.emf.ecore-2.37.0.jar (1.4 MB at 589 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/io/github/classgraph/classgraph/4.8.179/classgraph-4.8.179.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/ow2/asm/asm/9.7.1/asm-9.7.1.jar (126 kB at 52 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext/2.37.0/org.eclipse.xtext-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/apache/jena/jena-core/4.10.0/jena-core-4.10.0.jar (2.0 MB at 837 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib/2.37.0/org.eclipse.xtend.lib-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.common.types/2.37.0/org.eclipse.xtext.common.types-2.37.0.jar (526 kB at 204 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase.lib/2.37.0/org.eclipse.xtext.xbase.lib-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib/2.37.0/org.eclipse.xtend.lib-2.37.0.jar (137 kB at 53 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib.macro/2.37.0/org.eclipse.xtend.lib.macro-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/io/github/classgraph/classgraph/4.8.179/classgraph-4.8.179.jar (576 kB at 220 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.util/2.37.0/org.eclipse.xtext.util-2.37.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase.lib/2.37.0/org.eclipse.xtext.xbase.lib-2.37.0.jar (170 kB at 63 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/ch/qos/reload4j/reload4j/1.2.25/reload4j-1.2.25.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtend/org.eclipse.xtend.lib.macro/2.37.0/org.eclipse.xtend.lib.macro-2.37.0.jar (84 kB at 30 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.equinox.common/3.17.100/org.eclipse.equinox.common-3.17.100.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.util/2.37.0/org.eclipse.xtext.util-2.37.0.jar (327 kB at 116 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.osgi/3.18.300/org.eclipse.osgi-3.18.300.jar
Downloaded from central: https://repo.maven.apache.org/maven2/ch/qos/reload4j/reload4j/1.2.25/reload4j-1.2.25.jar (333 kB at 117 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/com/google/inject/guice/7.0.0/guice-7.0.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext.xbase/2.37.0/org.eclipse.xtext.xbase-2.37.0.jar (2.7 MB at 913 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/jakarta/inject/jakarta.inject-api/2.0.1/jakarta.inject-api-2.0.1.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.equinox.common/3.17.100/org.eclipse.equinox.common-3.17.100.jar (151 kB at 51 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/jakarta/inject/jakarta.inject-api/2.0.1/jakarta.inject-api-2.0.1.jar (11 kB at 3.5 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/com/google/inject/guice/7.0.0/guice-7.0.0.jar (802 kB at 257 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/xtext/org.eclipse.xtext/2.37.0/org.eclipse.xtext-2.37.0.jar (2.7 MB at 829 kB/s)
Downloaded from central: https://repo.maven.apache.org/maven2/org/eclipse/platform/org.eclipse.osgi/3.18.300/org.eclipse.osgi-3.18.300.jar (1.5 MB at 468 kB/s)
[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ lobid-resources ---
[INFO] Deleting /home/tobias/git/lobid-resources/target
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ lobid-resources ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 36 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.3:compile (default-compile) @ lobid-resources ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 18 source files to /home/tobias/git/lobid-resources/target/classes
[WARNING] system modules path not set in conjunction with -source 11
[INFO] /home/tobias/git/lobid-resources/src/main/java/org/lobid/resources/EtikettJson.java: Some input files use unchecked or unsafe operations.
[INFO] /home/tobias/git/lobid-resources/src/main/java/org/lobid/resources/EtikettJson.java: Recompile with -Xlint:unchecked for details.
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ lobid-resources ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] Copying 713 resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.3:testCompile (default-testCompile) @ lobid-resources ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 5 source files to /home/tobias/git/lobid-resources/target/test-classes
[WARNING] system modules path not set in conjunction with -source 11
[INFO] 
[INFO] --- maven-surefire-plugin:2.19.1:test (default-test) @ lobid-resources ---

-------------------------------------------------------
 T E S T S
-------------------------------------------------------
Running UnitTests
o.l.r.AlmaMarc21XmlToLobidJsonMetafixTest - Starting transforming File
d.h.l.h.EtikettMaker - use labels directory: /home/tobias/git/lobid-resources/target/classes/labels
d.h.l.h.EtikettMaker - Create labels....
d.h.l.h.EtikettMaker - ...succeed!
e.o.m.m.StreamBatchLogger - records processed: 10
e.o.m.m.StreamBatchLogger - records processed: 20
e.o.m.m.StreamBatchLogger - records processed: 30
e.o.m.m.StreamBatchLogger - records processed: 40
2026-04-28 15:28:50 INFO [org.elasticsearch.node.Node.<init>(Node.java:254)] - initializing ...
2026-04-28 15:28:50 INFO [org.elasticsearch.env.NodeEnvironment.maybeLogHeapDetails(NodeEnvironment.java:403)] - heap size [1.5gb], compressed ordinary object pointers [true]
2026-04-28 15:28:50 INFO [org.elasticsearch.node.Node.<init>(Node.java:282)] - node name [testNodeCgRvk], node ID [EjlVGLrVQj-ggfa4xkZaGw]
2026-04-28 15:28:50 INFO [org.elasticsearch.node.Node.<init>(Node.java:286)] - version[5.6.3], pid[147830], build[1a2f265/2017-10-06T20:33:39.012Z], OS[Linux/6.8.0-110-generic/amd64], JVM[Ubuntu/OpenJDK 64-Bit Server VM/17.0.18/17.0.18+8-Ubuntu-124.04.1]
2026-04-28 15:28:50 INFO [org.elasticsearch.node.Node.<init>(Node.java:299)] - JVM arguments [-Xmx1536m]
2026-04-28 15:28:50 INFO [org.elasticsearch.plugins.PluginsService.logPluginInfo(PluginsService.java:175)] - no modules loaded
2026-04-28 15:28:50 INFO [org.elasticsearch.plugins.PluginsService.logPluginInfo(PluginsService.java:178)] - loaded plugin [org.elasticsearch.transport.Netty4Plugin]
2026-04-28 15:28:51 INFO [org.elasticsearch.discovery.DiscoveryModule.<init>(DiscoveryModule.java:99)] - using discovery type [single-node]
2026-04-28 15:28:51 INFO [org.elasticsearch.node.Node.<init>(Node.java:515)] - initialized
2026-04-28 15:28:51 INFO [org.elasticsearch.node.Node.start(Node.java:656)] - starting ...
2026-04-28 15:28:52 INFO [org.elasticsearch.transport.TransportService.doStart(TransportService.java:212)] - publish_address {127.0.0.1:19201}, bound_addresses {127.0.0.1:19201}
2026-04-28 15:28:52 INFO [org.elasticsearch.cluster.service.ClusterService.publishAndApplyChanges(ClusterService.java:725)] - new_master {testNodeCgRvk}{EjlVGLrVQj-ggfa4xkZaGw}{UX6KLc4rRCeqv4Zo8uLwTQ}{127.0.0.1}{127.0.0.1:19201}, reason: single-node-start-initial-join[{testNodeCgRvk}{EjlVGLrVQj-ggfa4xkZaGw}{UX6KLc4rRCeqv4Zo8uLwTQ}{127.0.0.1}{127.0.0.1:19201}]
2026-04-28 15:28:52 INFO [org.elasticsearch.http.netty4.Netty4HttpServerTransport.doStart(Netty4HttpServerTransport.java:321)] - publish_address {127.0.0.1:19200}, bound_addresses {127.0.0.1:19200}
2026-04-28 15:28:52 INFO [org.elasticsearch.node.Node.start(Node.java:759)] - started
2026-04-28 15:28:52 INFO [org.elasticsearch.gateway.GatewayService$GatewayRecoveryListener$1.clusterStateProcessed(GatewayService.java:301)] - recovered [0] indices into cluster_state
o.l.r.CulturegraphXmlFilterHbzToJsonTest - Start extraction, transformation and creation of json bulk ... 
o.l.r.CulturegraphXmlFilterHbzToJsonTest - Done extraction, transformation and creation of json bulk
2026-04-28 15:28:52 INFO [org.elasticsearch.cluster.metadata.MetaDataCreateIndexService$1.execute(MetaDataCreateIndexService.java:454)] - [cg] creating index, cause [auto(bulk api)], templates [], shards [5]/[1], mappings []
2026-04-28 15:28:53 INFO [org.elasticsearch.cluster.metadata.MetaDataDeleteIndexService.deleteIndices(MetaDataDeleteIndexService.java:100)] - [cg/fFnmYfDvQyiuy8BNTiwdNg] deleting index
2026-04-28 15:28:53 INFO [org.elasticsearch.node.Node.stop(Node.java:769)] - stopping ...
2026-04-28 15:28:53 INFO [org.elasticsearch.node.Node.stop(Node.java:798)] - stopped
2026-04-28 15:28:53 INFO [org.elasticsearch.node.Node.close(Node.java:816)] - closing ...
2026-04-28 15:28:53 INFO [org.elasticsearch.node.Node.close(Node.java:888)] - closed
Tests run: 3, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 23.117 sec <<< FAILURE! - in UnitTests
transformFile(org.lobid.resources.AlmaMarc21XmlToLobidJsonMetafixTest)  Time elapsed: 19.198 sec  <<< ERROR!
java.lang.NoSuchMethodError: 'java.util.Set com.fasterxml.jackson.databind.JsonNode.properties()'
        at org.lobid.resources.AlmaMarc21XmlToLobidJsonMetafixTest.transformFile(AlmaMarc21XmlToLobidJsonMetafixTest.java:129)


Results :

Tests in error: 
  AlmaMarc21XmlToLobidJsonMetafixTest.transformFile:129 » NoSuchMethod 'java.uti...

Tests run: 3, Failures: 0, Errors: 1, Skipped: 0

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  45.436 s
[INFO] Finished at: 2026-04-28T15:28:53+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project lobid-resources: There are test failures.
[ERROR] 
[ERROR] Please refer to /home/tobias/git/lobid-resources/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
```

Same is true for web tests which break:

```
~/git/lobid-resources/web$ sbt "test:testOnly *IntegrationTest"
...
here are a lot of errors
...
[error] Test tests.IndexIntegrationTest.testResultCount[(132) {Q=dnbId:1045213144}] failed: java.lang.NoClassDefFoundError: javax/inject/Provider, took 0.002 sec
[error]     at play.api.inject.BindingKey$.apply(Binding.scala:62)
[error]     at play.api.inject.package$.bind(package.scala:44)
[error]     at play.api.test.FakeApplication.<init>(Fakes.scala:211)
[error]     at play.test.FakeApplication.<init>(FakeApplication.java:53)
[error]     at play.test.FakeApplication.<init>(FakeApplication.java:70)
[error]     at play.test.Helpers.fakeApplication(Helpers.java:118)
[error]     at tests.IndexIntegrationTest.testResultCount(IndexIntegrationTest.java:204)
[error]     at jdk.internal.reflect.GeneratedMethodAccessor3.invoke(Unknown Source)
[error]     at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[error]     at java.lang.reflect.Method.invoke(Method.java:569)
[error]     ...
[error] Caused by: java.lang.ClassNotFoundException: javax.inject.Provider
[error]     at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
[error]     at jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
[error]     at java.lang.ClassLoader.loadClass(ClassLoader.java:525)
[error]     ... 54 more
2026-04-28 15:40:19 INFO [org.elasticsearch.cluster.metadata.MetaDataDeleteIndexService.deleteIndices(MetaDataDeleteIndexService.java:100)] - [resources-alma-fix-staging/Yf5D5804T9ObK7Y-3tNvWQ] deleting index
2026-04-28 15:40:19 INFO [org.elasticsearch.node.Node.stop(Node.java:769)] - stopping ...
2026-04-28 15:40:19 INFO [org.elasticsearch.node.Node.stop(Node.java:798)] - stopped
2026-04-28 15:40:19 INFO [org.elasticsearch.node.Node.close(Node.java:816)] - closing ...
2026-04-28 15:40:19 INFO [org.elasticsearch.node.Node.close(Node.java:888)] - closed
[error] Failed: Total 172, Failed 172, Errors 0, Passed 0
[error] Failed tests:
[error]         tests.AcceptIntegrationTest
[error]         tests.IndexIntegrationTest
[error] (test:testOnly) sbt.TestsFailedException: Tests unsuccessful
[error] Total time: 109 s, completed 28.04.2026, 15:40:20
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -8,10 +8,10 @@ jobs:
       MAVEN_OPTS: "-Xmx768M"
     steps:
     - uses: actions/checkout@v2
-    - name: Set up JDK 11
+    - name: Set up JDK 17
       uses: actions/setup-java@v1
       with:
-        java-version: 11
+        java-version: 17
     - name: Cache Maven packages
       uses: actions/cache@v4
       with:
@@ -54,4 +54,4 @@ jobs:
     - name: Build and Test with play
       run: |
         cd web
-        sbt -v +test
+        sbt  --java-home $JAVA_HOME test

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -23,7 +23,8 @@ jobs:
         cd ..
         git clone https://github.com/metafacture/metafacture-core.git
         cd metafacture-core
-        git checkout master
+        git checkout -b 7a64c159a2e2dfc7d1673c83699254ad148f4011
+        git reset --hard 7a64c159a2e2dfc7d1673c83699254ad148f4011
         ./gradlew publishToMavenLocal
         cd -
     - name: Build with Maven

--- a/pom.xml
+++ b/pom.xml
@@ -15,17 +15,17 @@
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-io</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-files</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-json</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
             <exclusions>
                 <exclusion>
                     <groupId>commons-logging</groupId>
@@ -40,72 +40,72 @@
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-biblio</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-formeta</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-monitoring</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-strings</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-formatting</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-elasticsearch</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-csv</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-flowcontrol</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metamorph</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-plumbing</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metamorph-test</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-xml</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafacture-mangling</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
         </dependency>
         <dependency>
             <groupId>org.metafacture</groupId>
             <artifactId>metafix</artifactId>
-            <version>master-SNAPSHOT</version>
+            <version>7a64c159a2e2dfc7d1673c83699254ad148f4011-SNAPSHOT</version>
             <exclusions>
                 <exclusion>
                     <groupId>log4j</groupId>

--- a/web/build.sbt
+++ b/web/build.sbt
@@ -13,7 +13,7 @@ libraryDependencies ++= Seq(
   "com.typesafe.play" % "play-test_2.11" % "2.4.11",
   "org.apache.logging.log4j" % "log4j-core" % "2.9.1",
   "org.elasticsearch.plugin" % "parent-join-client" % "5.6.3",
-  "org.mockito" % "mockito-core" % "1.9.5" %Test,
+  "org.mockito" % "mockito-core" % "3.9.0" %Test,
   "com.google.gdata" % "core" % "1.47.1" exclude ("com.google.guava", "guava"),
   "org.easytesting" % "fest-assert" % "1.4" %Test,
   "org.xbib.elasticsearch.plugin" % "elasticsearch-plugin-bundle" % "5.4.1.0",
@@ -25,6 +25,8 @@ resolvers += "Local Maven Repository" at Path.userHome.asFile.toURI.toURL + ".m2
 lazy val root = (project in file(".")).enablePlugins(PlayJava)
 
 javacOptions ++= Seq("-source", "11", "-target", "11")
+javaOptions in (Test,run) += "--add-opens=java.base/java.lang=ALL-UNNAMED"
+javaOptions in (Test,run) += "--add-opens=java.base/java.util=ALL-UNNAMED"
 
 import com.typesafe.sbteclipse.core.EclipsePlugin.EclipseKeys
 

